### DISCUSSION TITLE
Contribute more than Places to OSM

### DISCUSSION THREAD

**Discussion by timoto** [organicmaps/organicmaps#8257] [Ideas]
As far as I can tell, I'm only able to add or edit a Place (POI) to contribute to OSM via OrganicMaps.

Would it be possible to contribute further, in particular walking tracks for example ?

**Comment by biodranik** [comment]
After track recorder #8215 is implemented, uploading to OSM can be added.

**Reply by matkoniecz** [reply]
Though raw tracks cannot be really added as road/paths with raw unchanged geometry... And there is a lot of fun with intersecting existing lines (power lines, roads, paths, rivers...)
